mhkit/dolfyn/adp/clean.py

```diff
@@ -372,31 +390,47 @@ def remove_surface_interference(
     # Get all variables with 'range' coordinate
     profile_vars = [h for h in ds.keys() if any(s for s in ds[h].dims if "range" in s)]
 
-    # Surface interference distance
+    # Fetch cell size
+    cs = [
+        a
+        for a in ds.attrs
+        if (
+            ("cell_size" in a)
+            and ("_bt" not in a)
+            and ("_alt" not in a)
+            and ("wave" not in a)
+        )
+    ]
+
```

Comment:
Should this: 
`raise KeyError("No valid 'cell_size' attribute found in dataset.")`

Or will there always be a "cell_size"?

Reply:
There will always be a "cell_size", yes.